### ISSUE TITLE
migration of ingress do not work 

### ISSUE THREAD

**Issue by fragtom** [containous/traefik-migration-tool#11]
Promising tool for migration did not work on my ingresses. So, there we have a k8s distribution, where the previously used traefik 1.7~ version should be migrated. 

```bash
docker run --rm -v ${PWD}:/data containous/traefik-migration-tool ingress --input /data/in --output /data/out 
```

Produces output, that is identically to input:

```yml
apiVersion: v1
items:
- apiVersion: extensions/v1beta1
  kind: Ingress
  metadata:
    annotations:
      ingress.kubernetes.io/auth-secret: basic-auth-document-db
      ingress.kubernetes.io/auth-type: basic
      ingress.kubernetes.io/custom-request-headers: XYZ-My-Custom-Header:dev
      kubernetes.io/ingress.class: traefik
    creationTimestamp: "2018-02-27T10:03:59Z"
    generation: 9
    name: dev-protected
    namespace: dev
    resourceVersion: "277178439"
    selfLink: /apis/extensions/v1beta1/namespaces/dev/ingresses/dev-protected
    uid: 87d8d657-1ba5-11e8-a9cd-06fa2d724cac
  spec:
    rules:
    - host: svc.test.migrate.traefik.to.v2.com
      http:
        paths:
        - backend:
            serviceName: v2
            servicePort: 80
```

What could be the trick?

**Comment by ldez** [comment]
hello,

It's related to `items`, currently the migration tool don't support that.

If you extract you ingress into files with the following format, the migration will work.
```yml
apiVersion: extensions/v1beta1
kind: Ingress

#...
```

**Comment by ldez** [comment]
When I try to use your file, I have an error:
```
error while decoding YAML object. Err was: Object 'Kind' is missing in 'apiVersion: v1
...
```

**Comment by ldez** [comment]
Fixed by a41334769ee0776dcf90730e35071a20933ef146
